### ISSUE TITLE
Implement Bonus Round rules for missing enemies

### ISSUE THREAD

**Issue by onetruepurple** [TerryCavanagh/diceydungeons.com#1207]
- Gardener
- Magician
- Dryad
- Marshmallow
- Stereohead
- Keymaster
- Yeti
- Aurora
- Crystalina
- Rhino Beetle
- Wicker Man
- Gargoyle
- Aoife
- Audrey
- Beatrice
- (human Jester)
- (D**** M*****)

**Comment by onetruepurple** [comment]
Consider filling this out before Reunion.

**Comment by TerryCavanagh** [comment]
Yep - this is something I'd really like to take another look at.

(fyi list is missing Copycat, and I'm not going to do human Jester)
